packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -130,6 +130,152 @@ export class GaussianSplattingMesh extends Mesh {
         return super.render(subMesh, enableAlphaMode, effectiveMeshReplacement);
     }
 
+    /**
+     * Code from https://github.com/dylanebert/gsplat.js/blob/main/src/loaders/PLYLoader.ts Under MIT license
+     * Converts a .ply data array buffer to splat
+     * if data array buffer is not ply, returns the original buffer
+     * @param data the .ply data to load
+     * @returns the loaded splat buffer
+     */
+    public static ConvertPLYToSplat(data: any): ArrayBuffer {

```

Comment:
`ArrayBuffer` would be better than `any`.

Reply:
Fixed.